pages/article/ArticleHandler.inc.php

```diff
@@ -357,8 +422,9 @@ function cite($args, $request) {
 		$router = $request->getRouter();
 		$this->setupTemplate($request);
 		$articleId = isset($args[0]) ? $args[0] : 0;
-		$citeType = isset($args[1]) ? $args[1] : null;
-		$returnFormat = isset($args[2]) ? $args[2] : null;
+		$version = isset($args[1]) ? $args[1] : null;
+		$citeType = isset($args[2]) ? $args[2] : null;
```

Comment:
this is probably OK, because these old URLs are not so important as the "download" URLs